auto_update_scheduled.yaml

```
@@ -500,631 +500,638 @@
+                states.update
+                | selectattr('state','eq','on')
+                | selectattr('entity_id', 'in', pending_updates_list)
+                | rejectattr('entity_id', 'in', input_update_exclusions)
+                | list
+                | count
+                | int(0)
+              ) < 1
+            }}
+          - condition: and
+            conditions:
+            - condition: trigger
```

yamllint (1.35.1)

[warning] wrong indentation: expected 14 but found 12

(indentation)